DetectorDescription/Core/test/DDName.cppunit.cc

```diff
@@ -0,0 +1,54 @@
+#include <cppunit/extensions/HelperMacros.h>
+
+#include "DetectorDescription/Core/interface/DDName.h"
+#include "DetectorDescription/Core/interface/Singleton.h"
+#include "cppunit/TestAssert.h"
+#include "cppunit/TestFixture.h"
+
+class testDDName : public CppUnit::TestFixture {
+  CPPUNIT_TEST_SUITE(testDDName);
+  CPPUNIT_TEST(checkNames);
+  CPPUNIT_TEST_SUITE_END();
+public:
+  void setUp() override{}
+  void tearDown() override {}
+  void buildIt();
+  void testloading();
+  void checkNames();
+};
+
+CPPUNIT_TEST_SUITE_REGISTRATION(testDDName);
+
+void
+testDDName::buildIt() {
+  DDName name;
+  DDName anotherName;
+  std::cerr << "\nDDName name=" << name << " id=" << name.id()
+	    << ", another name=" << anotherName << " id=" << anotherName.id();
+  int a[100000]{};
```

Comment:
This should just be uninitialized values. Is that what you want?

Reply:
Doing
```
int a[100000] = {};
```
Will zero initialize all the elements.

Reply:
I misread one of the web sites info. Testing using 'compiler explorer' does show that what you did will initialize all values to 0.  Sorry for the incorrect comment.